client.go

```diff
@@ -40,6 +41,7 @@ type Client struct {
 	ReadTimeout    time.Duration     // net.Conn.SetReadTimeout value for connections, defaults to 2 seconds - overridden by Timeout when that value is non-zero
 	WriteTimeout   time.Duration     // net.Conn.SetWriteTimeout value for connections, defaults to 2 seconds - overridden by Timeout when that value is non-zero
 	TsigSecret     map[string]string // secret(s) for Tsig map[<zonename>]<base64 secret>, zonename must be in canonical form (lowercase, fqdn, see RFC 4034 Section 6.2)
+	TsigVerifySkip bool              // Skip verifying a TSIG RR immediately at read time
```

Comment:
Shouldn't a `nil` TsigSecret map convey the same info?

Reply:
I thought about that and liked it because it's very similar to the logic that is in the Write() call.  But as I mentioned in the PR description I was trying to do it in a manner that doesn't change the default behavior.  By default today if this is nil and you receive a TSIG, that message will not be processed - it's immediately rejected before "serving" it.  If we change the logic as you suggest, that would now be received and processed.

I am fine to change it that way if you are okay changing the default behavior of the package.

Reply:
I think *not* adding this boolean is cleaner, chaning the default behavior is the price we then pay. So be it.